sdk/php/dev/src/PhpSdkDev.php

```diff
@@ -16,7 +16,16 @@
 #[DaggerObject]
 final class PhpSdkDev
 {
-    private const SDK_ROOT='/src/sdk/php';
+    private const SDK_ROOT = '/src/sdk/php';
+
+    public $base_container;
+
+    #[DaggerFunction]
+    public function __construct(
+        public ?Container $container = null,
+    ) {
+        $this->base_container = $container;
+    }
```

Comment:
```suggestion
    #[DaggerFunction]
    public function __construct(
        private ?Container $baseContainer = null,
    ) {
    }
```

This would assign it as a private property that can only be null or a Container.

PHP variables are usually in `camelCase` so `$baseContainer` is preferable.

Keep the property `private` unless we actually need access to it outside of the class.